### PR TITLE
Allow setting authentication header on External channels

### PR DESCRIPTION
**Pull request by norkans7**
Addresses https://github.com/rapidpro/rapidpro/issues/1174

### CODE REVIEW

File: temba/channels/models.py [hunk 1]
```diff
@@ -288,6 +288,7 @@ class Channel(TembaModel):
     CONFIG_MESSAGING_SERVICE_SID = "messaging_service_sid"
     CONFIG_MAX_CONCURRENT_EVENTS = "max_concurrent_events"
     CONFIG_ALLOW_INTERNATIONAL = "allow_international"
+    CONFIG_SEND_AUTHORIZATION = "send_authorization"
```
Comment (nicpottier) [inline temba/channels/models.py]: This should be on the external channel type.

File: temba/channels/types/external/views.py [hunk 2]
```diff
@@ -73,6 +73,13 @@ class ClaimForm(ClaimViewMixin.Form):
             ),
         )
 
+        send_authorization = forms.CharField(
+            max_length=2048,
+            label=_("Authorization Header Value"),
+            required=False,
+            help_text=_("The Authorization header value added when calling the URL"),
```
Comment (nicpottier) [inline temba/channels/types/external/views.py]: `"The Authorization header value added when calling the URL (if any)"`

File: locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1361,6 +1361,16 @@ msgstr "El token secreto que este transmisor deberá usar cuando firme los pedid
...
 
+#, fuzzy
+#| msgid "Numeric Value:"
+msgid "Authorization Header Value"
+msgstr "Valor numérico:"
+
+#, fuzzy
+#| msgid "What HTTP method to use when calling the URL"
+msgid "The Authorization header value added when calling the URL"
+msgstr "Qué método de HTTP utiliza cuando llama al URL"
```
Comment (rowanseymour) [inline locale/es/LC_MESSAGES/django.po]: side note.. these aren't right.. wonder if gettext ignores fuzzy stuff?

Comment (nicpottier) [reply locale/es/LC_MESSAGES/django.po]: To my understanding it does, fuzzy's are just "suggestions" to the translators.

File: temba/channels/models.py (hunk 1 above)
Comment (norkans7) [reply temba/channels/models.py]: This is just a class constant, you mean to move it to `ChannelType` instead?

Comment (nicpottier) [reply temba/channels/models.py]: Yes. Constants that are channel specific belong on the channel type.

Comment (norkans7) [reply temba/channels/models.py]: in a channel type package, type module import from views module, so w e won't bi able to import from type in views 
one way I think for that to work is to add a new module like const.py that will be containing the constants. Is that what you request?

Comment (norkans7) [reply temba/channels/models.py]: I can add that and change the other const used just by one channel type too

File: locale/es/LC_MESSAGES/django.po (hunk 3 above)
Comment (rowanseymour) [reply locale/es/LC_MESSAGES/django.po]: Dug into this a bit to make sure - seems it's `compilemessages` that strips these out - it has a param `--use-fuzzy` if you wanted them to end up in the mo files.

File: temba/channels/models.py (hunk 1 above)
Comment (nicpottier) [reply temba/channels/models.py]: It's a local import in the view, see WhatsApp channel.

**Pull request merged**